ResearchKit/Consent/ORKVisualConsentStepViewController.m

```diff
@@ -661,13 +662,12 @@ - (NSUInteger)indexOfViewController:(UIViewController *)viewController {
         return NSNotFound;
     }
     
-    NSUInteger index = NSNotFound;
     for (NSNumber *key in _viewControllers) {
         if (_viewControllers[key] == viewController) {
-            index = [key unsignedIntegerValue];
+            return [key unsignedIntegerValue];
```

Comment:
That's fair enough!

However, again, I don't think that exit points in the middle of methods is good style, as could be error prone. This method is simple enough, but with longer ones they can lead to "*code blindness*".

I prefer that all method exit points are either at the top (because of initial conditions, like the first one here), or at the bottom. That way you can follow the flow of what the function does easily.

I think legibility is more important than performance in this case (and probably the compiler would optimize the extra variable away).

@YuanZhu-apple decides. ;-)